README.md

```diff
@@ -469,6 +499,10 @@ Usage may look like this
 
     # Deprecated, see https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/enabling-debug-logging
     VERBOSE: false
+
+    # The specific activity that should be taken on this event, see
+    # [Determining the mode](#determining-the-mode) above.
+    ACTIVITY: ""
```

Comment:
Who turns `""` into `None` ? I think there should be some code somewhere doing that, and I'm not seeing it. Am I missing something or is it missing ?